polaris-core/src/main/java/io/polaris/core/PolarisConfigurationStore.java

```diff
@@ -103,10 +101,17 @@ public interface PolarisConfigurationStore {
       PolarisConfiguration<T> config) {
     if (config.hasCatalogConfig()
         && catalogEntity.getPropertiesAsMap().containsKey(config.catalogConfig())) {
-      LOGGER.debug("Loaded config from catalog: {}", config.catalogConfig());
+      logger().debug("Loaded config from catalog: {}", config.catalogConfig());
```

Comment:
We may just remove the debug log, as it logs every time loading one single value. cc @eric-maynard @collado-mike

Reply:
thanks, removed

Reply:
I'm not sure I agree that this log is useless

Reply:
I'd not consider it's useless, I think it's not worth spending extra lines of code on it. Moreover, there could be too many logs once we turn on the debug level, as it logs for each single value. But I'm OK with either keeping it or not.

Reply:
I think it's fine to remove altogether. It may make more sense for implementations to log when values are loaded/changed